.golangci.yml

```diff
@@ -0,0 +1,46 @@
+run:
+  # timeout for analysis
+  deadline: 4m
+
+  # Linting uses a lot of memory. Keep it under control by only running a single
+  # worker.
+  concurrency: 1
+
+linters-settings:
+  govet:
+    # Don't report about shadowed variables
+    check-shadowing: false
```

Comment:
check  if we can remove.

Reply:
I removed it and ran it for the whole git history. Didn't mention any new issues so we can use the configuration without this exception.